test/generic/test_object_storage.py

```diff
@@ -4,24 +4,32 @@
 
 
 def test_clear_objects():
+    """
+    Asserts that objects in storage are cleared
+    """
     obj_storage = object_storage.ObjectStorage()
-
+    """
+    obj_storage is a wrapper object to a collection of objects 
+    """
     x = torch.tensor(1)
     obj_storage.set_obj(x)
 
-    objs = obj_storage.current_objects()
+    objs = obj_storage.current_objects()#Returns a copy of the objects in obj_storage(here:x)
```

Comment:
[`black`](https://github.com/psf/black) requires comments to begin two spaces after code. I think this is why the tests are failing

Reply:
Oh, I'll fix that 
Thank you